dagmc/test/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ list(APPEND LINK_LIBS ${DAG_LINK_LIBS})
 # Include directories
 include_directories(${MOAB_INCLUDE_DIRS})
 include_directories(${CMAKE_SOURCE_DIR}/dagmc)
+include_directories(${CMAKE_BINARY_DIR}/dagmc)
```

Comment:
Why is this dir necessary now? Or was it always and just missing?

Reply:
This is for the new `DagMCVersion.h` file which gets configured when running CMake. We need to be able to find the configured file in the build directory.